Scripts/ScreenshotUtils/CameraExtensions.cs

```diff
@@ -13,17 +14,21 @@ public static class CameraExtensions
 		/// <param name="camera">The camera to use</param>
 		/// <param name="width">The width of the texture</param>
 		/// <param name="height">The height of the texture</param>
-		/// <param name="textureFormat">The texture format</param>
 		/// <param name="cullingMask">The culling mask used during the render</param>
+		/// <param name="textureFormat">The Texture format</param>
+		/// <param name="renderTextureFormat">The Render Texture format</param>
 		/// <returns>A Texture2D object of the given width/height</returns>
-		public static Texture2D RenderToTexture(this Camera camera, int width, int height,
-			TextureFormat textureFormat = TextureFormat.ARGB32, int cullingMask = 0)
+		public static Texture2D RenderToTexture(this Camera camera, int width, int height, int cullingMask = 0,
+			TextureFormat textureFormat = TextureFormat.ARGB32,
+			RenderTextureFormat renderTextureFormat = RenderTextureFormat.ARGBFloat,
```

Comment:
I don't think we should have this value as default. `RenderTextureFormat.ARGBFloat`. 

I think we should keep the default at what it was before we made this change which is `RenderTextureFormat.Default`. 
The docs state:
> Default color render texture format: will be chosen accordingly to Frame Buffer format and Platform.

Anybody who used this previously and never specified the `RenderTextureFormat` will now get a different default value and can result in different behaviour. 

In our project that different behaviour was a fatal crash (with no stack-trace) since we used the util at runtime.

Reply:
EZ FIX

Reply:
And don't forget it has an x8 MSAA by default... be careful when using it on runtime.